Classes/Utility/AttributeUtility.php

```diff
@@ -41,6 +41,58 @@ public static function findAttribute(int $attributeId, string $selectFields = '*
         );
     }
 
+    /**
+     * Returns all attribute id's for corresponding product.
+     *
+     * @param int $productId
+     * @param string $selectFields
+     * @return array|null
+     */
+    public static function findAllAttributesForProduct(
```

Comment:
I think this works better without "All" in there. Unnecessary? There's `findAllAttributes` too, but refactoring to `findAttributes` seems like a good idea. Agree?

```suggestion
    public static function findAttributesForProduct(
```